torch/csrc/jit/serialization/export_bytecode.cpp

```diff
@@ -137,7 +137,7 @@ std::unique_ptr<mobile::Code> compileGraphToMobileCode(
     const std::string& name,
     const std::shared_ptr<Graph>& graph,
     const CompilationOptions& compilation_options,
-    BackendDebugInfoRecorder& debug_info_recorder) {
+    BackendDebugInfoRecorder* debug_info_recorder) {
```

Comment:
Don't make this optional.

The caller can already make it optional by passing one and not using.

Reply:
Hmm do you mean just initialize BackendDebugInfoRecorder in the script_init?

Reply:
yeah just declare a new one with `BackendDebugInfoRecorder recorder` and pass it along.